forest/sycamore/base.py

```diff
@@ -1,23 +1,28 @@
+import datetime
+import logging
 import os
 from typing import Optional, List
-import logging
 
-from forest.sycamore.responses import agg_changed_answers_summary
 from forest.sycamore.common import (aggregate_surveys_config,
                                     aggregate_surveys_no_config)
+from forest.sycamore.responses import (agg_changed_answers_summary,
+                                       responses_by_submission)
 from forest.sycamore.submits import (survey_submits,
-                                     survey_submits_no_config,
-                                     get_all_interventions_dict)
+                                     survey_submits_no_config)
 
 logger = logging.getLogger(__name__)
 
+EARLIEST_DATE = "2000-01-01"
+TODAY_MIDNIGHT = datetime.datetime.today().strftime("%Y-%m-%d")+"T23:59:00"
+
 
 def survey_stats_main(
         study_folder: str, output_folder: str, tz_str: str = "UTC",
         participant_ids: Optional[List] = None,
-        time_start: Optional[str] = None, time_end: Optional[str] = None,
+        time_start: str = EARLIEST_DATE, time_end: str = TODAY_MIDNIGHT,
```

Comment:
I defined EARLIEST_DATE because I wanted the same default date to be used in multiple functions. It just needs to be earlier than any Beiwe data could have been collected.